birdy/client/base.py

```diff
@@ -344,9 +344,13 @@ def sort_inputs_key(i):
     ----------
     i: owslib.wps.Input
       An owslib Input
+
+    Notes
+    -----
+    The defaultValue for ComplexData is ComplexData instance specifying mimetype, encoding and schema.
     """
     conditions = [
-        i.minOccurs >= 1 and i.defaultValue is None,
+        i.minOccurs >= 1 and (i.defaultValue is None or isinstance(i.defaultValue, ComplexData)),
```

Comment:
`... or isinstance(i, ComplexDataInput)`

see here:
https://github.com/geopython/OWSLib/blob/f48d8f69ada2a746072b16c8b3cba1b62c5fd1ec/owslib/wps.py#L178

Reply:
in case of ComplexInput checking the type of the input ... not the defaultValue type.

Reply:
Hum, this doesn't seem to work. The input is an `wps.Input` object, and the default value a `wps.ComplexData` object. So `is_complexdata` does not work on either.

Reply:
Yes, you are right. The names (`Input`, `ComplexDataInput`) are just confusing :)